modules/serverless-installing-cli-linux-ibm-power-tarball.adoc

```diff
@@ -0,0 +1,35 @@
+[id="installing-cli-linux-ibm-power-tarball_{context}"]
+= Installing the Knative CLI for Linux on IBM Power Systems
+
+For Linux distributions, you can download the `kn` CLI directly as a `tar.gz` archive.
+
+.Procedure
+
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest[`kn` CLI].
+
+. Unpack the archive:
++
+[source,terminal]
+----
+$ tar -xf <file>
+----
+
+. Move the `kn` binary to a directory on your PATH.
+
+. To check your path, run:
++
+[source,terminal]
+----
+$ echo $PATH
+----
++
+[NOTE]
+====
+If you do not use {op-system-base} or {op-system-base}, ensure that *libc* is installed in a directory on your library path.
```

Comment:
Same questions about this note (double {op-system-base}, whether it's meant to be part of the last step or not) as for the Z version of this topic.